test/e2e/feature/feature.go

```diff
@@ -118,6 +118,18 @@ var (
 	// OWNER: sig-node
 	// Testing downward API huge pages
 	DownwardAPIHugePages = framework.WithFeature(framework.ValidFeatures.Add("DownwardAPIHugePages"))
+	// owning-sig: sig-node
+	// kep: https://kep.k8s.io/4816
+	// test-infra jobs:
+	// - "dra-alpha" in https://testgrid.k8s.io/sig-node-dynamic-resource-allocation
```

Comment:
```suggestion
	// - "ci-kind-dra-all" in https://testgrid.k8s.io/sig-node-dynamic-resource-allocation
```

I'm not seeing any such tests in https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/128586/pull-kubernetes-kind-dra-all/1884325562575491072 for this PR.

Ah, because of:
```
+ hack/ginkgo-e2e.sh '-ginkgo.label-filter=Feature: containsAny DynamicResourceAllocation && Feature: isSubsetOf { Alpha, Beta, DynamicResourceAllocation, DRAAdminAccess} && !Flaky && !Slow'
```

I'll update the job.

Reply:
-> https://github.com/kubernetes/test-infra/pull/34231

Reply:
Updated the PR to reference to correct job.

Reply:
@pohly 
https://github.com/kubernetes/test-infra/pull/34231 runs the tests for the correct features, but it still doesn't enable the correct features in kind. From the test run:

```
+ echo 'Enabling DRA feature(s): DRAAdminAccess DRAPrioritizedList DRAResourceClaimDeviceStatus.'
Enabling DRA feature(s): DRAAdminAccess DRAPrioritizedList DRAResourceClaimDeviceStatus.
+ kind create cluster --retain --config /dev/fd/63 --image dra/node:latest
++ cat test/e2e/dra/kind.yaml
++ for feature in ${features}
++ echo '  DRAAdminAccess: true'
Creating cluster "kind" ...
```

I have created https://github.com/kubernetes/test-infra/pull/34242 that I think should fix this. It is very similar to https://github.com/kubernetes/test-infra/pull/34231